### DISCUSSION TITLE
Welcome to material-table Discussions!

### DISCUSSION THREAD

**Discussion by matthewoestreich** [material-table-core/core#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by haroldwaste** [comment]
Hi @oze4 thank you for the initiative. I use material-table and would be very glad to switch to this project. I wonder if you plan to tackle the performance issues that material-table has (virtualization, memoization of rows, select rows without rerendering the whole table). I would be glad to help a bit and it would be a game changer feature. Thank you.

**Reply by matthewoestreich** [reply]
Hello! We do plan on refactoring as well. The more help we can get the better!

**Comment by artdude543** [comment]
Hi @oze4 nice to see this module is getting some love to get fixes/additions added etc... What are the plans to consider upgrading to the new material? Or to consider running a parallel version in (beta/alpha) using the latest alpha MUI build(s)? With the move to React 17 and MUI switching to that version on the `5.0.0-alpha` versions. Some have started to the switch over, the module works fine (to my quick testing) but would be amazing to see some support.

I'm aware of another fork which is using the 5 builds. But that's a fork of the main repo and not this one which contains your various changes/fixes.

Would love to hear your thoughts on this, thanks!

**Reply by matthewoestreich** [reply]
We are indeed working on a refactor over at https://github.com/material-table-core/next 

Do these changes break material table? If so, what should we change? Let me know how I can help test, etc...

**Reply by artdude543** [reply]
As far as I'm aware nothing "major" changed which breaks support in Material Table. Other than some changes to import names etc... but I've not seen anything "visual" break. Currently using this [fork](https://github.com/villuv/material-table/tree/feature/mui5). Which you can see the commits on what's needed etc... Pickers have now been merged into the main code for MUI so that external dependency isn't needed anymore. So lot's of benefits with the MUI 5 plus more to come in the pipeline!
